src/GNDStk/v1.9/containers/Axes/src/custom.hpp

```diff
@@ -0,0 +1,30 @@
+private:
+
+  static inline helpMap help = {};
+
+  /**
+   *  @brief Custom construct function
+   *
+   +  Sort the elements by index so that the indices are in order
+   */
+  void construct() {
+
+    // useful lambdas
+    auto getIndex = [] ( const auto& variant )
+                       { return variant.index().value(); };
+
+    std::sort( this->content.choice.begin(), this->content.choice.end(),
```

Comment:
As I mentioned in our phone conversation, sorting by index is probably something I should do automatically. I think that the Component base class could be made to detect the situation when it exists.

Reply:
Yes, I agree. This one can be generated automagically since we can deduce the situation.